packages/paste-website/src/components/homepage/GetStarted.tsx

```diff
@@ -1,25 +1,38 @@
 import * as React from 'react';
-import {Box} from '@twilio-paste/core/box';
-import {Grid, Column} from '@twilio-paste/core/grid';
+import {Box} from '@twilio-paste/box';
+import {Grid, Column} from '@twilio-paste/grid';
+import {Text} from '@twilio-paste/text';
 import {GetStarterWhy} from './GetStartedWhy';
 import {GetStartedInclusive} from './GetStartedInclusive';
 import {GetStartedRunning} from './GetStartedRunning';
 import {SITE_CONTENT_MAX_WIDTH} from '../../constants';
 
 export const GetStarted: React.FC = () => {
   return (
-    <Box maxWidth={SITE_CONTENT_MAX_WIDTH} marginLeft="auto" marginRight="auto">
-      <Grid as="section" gutter="space40" vertical={[true, false, false]}>
-        <Column span={4}>
-          <GetStarterWhy />
-        </Column>
-        <Column span={4}>
-          <GetStartedInclusive />
-        </Column>
-        <Column span={4}>
-          <GetStartedRunning />
-        </Column>
-      </Grid>
+    <Box backgroundColor="colorBackgroundPrimaryDarker" paddingX={['space90', 'space180']}>
+      <Box maxWidth={SITE_CONTENT_MAX_WIDTH} marginLeft="auto" marginRight="auto">
+        <Text
```

Comment:
this text is no longer part of the copy. https://docs.google.com/document/d/1-BaZ034vXyYMkdUDb_W3nExL111glUb2_7UQ7GldEhE/edit

Reply:
Got it. I missed that doc. Thanks.